internal/app/tfsec/parser/attribute.go

```diff
@@ -48,3 +52,66 @@ func (attr *Attribute) Range() Range {
 func (attr *Attribute) Name() string {
 	return attr.hclAttribute.Name
 }
+
+func (attr *Attribute) Contains(checkValue string) bool {
+	val := attr.Value()
+	if val.IsNull() {
+		return false
+	}
+	if val.Type().IsObjectType() || val.Type().IsMapType() {
+		valueMap := val.AsValueMap()
+		for key := range valueMap {
+			if key == checkValue {
+				return true
+			}
+		}
+		return false
+	}
+	if val.Type().IsListType() || val.Type().IsTupleType() {
+		valueSlice := val.AsValueSlice()
+		for _, value := range valueSlice {
+			if value.AsString() == checkValue {
+				return true
+			}
+		}
+		return false
+	}
+	return strings.Contains(val.AsString(), checkValue)
+}
+
+func (attr *Attribute) StartsWith(prefix string) bool {
+	if attr.Value().Type() == cty.String {
+		return strings.HasPrefix(attr.Value().AsString(), prefix)
+	}
+	return false
+}
+
+func (attr *Attribute) EndsWith(prefix string) bool {
+	if attr.Value().Type() == cty.String {
+		return strings.HasSuffix(attr.Value().AsString(), prefix)
+	}
+	return false
+}
+
+func (attr *Attribute) Equals(checkValue string) bool {
```

Comment:
We could potentially rename this to `EqualsStr` or similar, in case we also end up having `EqualsInt`? Alternatively we could have `Equals(checkValue interface{})` but that feels a bit gross. WDYT?

Reply:
Oh nevermind, I just read the `if attr.Value().Type() == cty.Number {` bit, so this handles everything - looks good :+1: